operations/jsonnet/lib/cortex/cortex.yaml

```diff
@@ -0,0 +1,74 @@
+# Configuration for running Cortex in single-process mode.
```

Comment:
Let's move this file into the acceptance test directory instead, since that's the only place it's being used